lib/morph/database.rb

```diff
@@ -128,7 +128,7 @@ def sql_query_safe(query, readonly = true)
 
     # Returns 0 if table doesn't exists (or there is some other problem)
     def no_rows(table = table_names.first)
-      q = sql_query_safe("select count(*) from '#{table}'")
+      q = sql_query_safe("select count(*) from \"#{table}\"")
```

Comment:
Rather than using backslashes to escape the double quote I think this would be more readable if you use % notation. That would make this line:

```ruby
q = sql_query_safe(%{select count(*) from "#{table}"}) 
```